app/controllers/search/agendaitems.js

```diff
@@ -105,7 +105,7 @@ export default class AgendaitemsSearchController extends Controller {
   navigateToAgendaitem(searchEntry, clickEvent) {
     if (searchEntry.meetingId) {
       // Check if we clicked an emphasis inside a linkTo or a linkTo
-      if (clickEvent?.target.parentElement.className.indexOf('card-link') > -1 || clickEvent?.target.className.indexOf('card-link') > -1) {
+      if (clickEvent?.target.parentElement.className.indexOf('l-result-card__link') > -1 || clickEvent?.target.className.indexOf('l-result-card__link') > -1) {
         // do nothing, this was a clicked link in the card and the router will transition later
         return;
```

Comment:
I testes this locally, could it be that this behaviour has changed? (maybe part due to ember upgrade)
What it should do:
When clicking the card you enter the else part, no link was clicked.
When clicking the link inside the card you would transition twice, once with the LinkTo, once with our `router.transitionTo` so you would get double entries in history.
Now however, when debugging both the if and else, I never hit the return in the if statement when clicking the link. But I also don't hit transitionTo. I only hit that when not clicking a link.

Also a possible issue.
When clicking the link in the card we also stopped hitting our plausible tracking event.
The behaviour on ` @onClickRow={{this.resultClicked}}` seems to have changed.
I think this is besides that styling changing here and should be looked at independently